ocdaction/tasks/models.py

```diff
@@ -42,79 +42,54 @@ class AnxietyScore(models.Model):
     SCORE_TEN = '10'
 
     ANXIETY_SCORE_CHOICES = (
-        (SCORE_ZERO, 'Zero'),
-        (SCORE_ONE, 'One'),
-        (SCORE_TWO, 'Two'),
-        (SCORE_THREE, 'Three'),
-        (SCORE_FOUR, 'Four'),
-        (SCORE_FIVE, 'Five'),
-        (SCORE_SIX, 'Six'),
-        (SCORE_SEVEN, 'Seven'),
-        (SCORE_EIGHT, 'Eight'),
-        (SCORE_NINE, 'Nine'),
-        (SCORE_TEN, 'Ten'),
+        (SCORE_ZERO, '0'),
+        (SCORE_ONE, '1'),
+        (SCORE_TWO, '2'),
+        (SCORE_THREE, '3'),
+        (SCORE_FOUR, '4'),
+        (SCORE_FIVE, '5'),
+        (SCORE_SIX, '6'),
+        (SCORE_SEVEN, '7'),
+        (SCORE_EIGHT, '8'),
+        (SCORE_NINE, '9'),
+        (SCORE_TEN, '10'),
     )
-    score = models.CharField(
+
+    score_after_0_min = models.CharField(
         max_length=2,
         choices=ANXIETY_SCORE_CHOICES,
         default=SCORE_ZERO
     )
-
-    task = models.ForeignKey('tasks.Task', on_delete=models.CASCADE)
-
-    def __unicode__(self):
-        return self.score
-
-    def user_name(self):
-        return self.task.user
-
-
-class AnxietyScoreCard(models.Model):
-    task = models.ForeignKey('Task', on_delete=models.CASCADE)
-    score_after_0_min = models.ForeignKey(
-        AnxietyScore,
-        blank=True,
-        null=True,
-        related_name='score_0',
-        on_delete=models.CASCADE
-    )
-
-    score_after_5_min = models.ForeignKey(
-        AnxietyScore,
-        blank=True,
-        null=True,
-        related_name='score_5',
-        on_delete=models.CASCADE
+    score_after_5_min = models.CharField(
+        max_length=2,
+        choices=ANXIETY_SCORE_CHOICES,
+        default=SCORE_ZERO
     )
-    score_after_10_min = models.ForeignKey(
-        AnxietyScore,
-        blank=True,
-        null=True,
-        related_name='score_10',
-        on_delete=models.CASCADE
+    score_after_10_min = models.CharField(
+        max_length=2,
+        choices=ANXIETY_SCORE_CHOICES,
+        default=SCORE_ZERO
     )
-    score_after_15_min = models.ForeignKey(
-        AnxietyScore,
-        blank=True,
-        null=True,
-        related_name='score_15',
-        on_delete=models.CASCADE
+    score_after_15_min = models.CharField(
+        max_length=2,
+        choices=ANXIETY_SCORE_CHOICES,
+        default=SCORE_ZERO
     )
-    score_after_30_min = models.ForeignKey(
-        AnxietyScore,
-        blank=True,
-        null=True,
-        related_name='score_30',
-        on_delete=models.CASCADE
+    score_after_30_min = models.CharField(
+        max_length=2,
+        choices=ANXIETY_SCORE_CHOICES,
+        default=SCORE_ZERO
     )
-
-    score_after_60_min = models.ForeignKey(
-        AnxietyScore,
-        blank=True,
-        null=True,
-        related_name='score_60',
-        on_delete=models.CASCADE
+    score_after_60_min = models.CharField(
+        max_length=2,
+        choices=ANXIETY_SCORE_CHOICES,
+        default=SCORE_ZERO
```

Comment:
I feel like 0 shouldn't be a default as it is the best score you can have aka No Anxiety, perhaps a 5 or even blank would be better so we can always tell when a user has not entered the score. In fact blank is best